moduleframework/mtf_scheduler.py

```diff
@@ -248,6 +250,35 @@ def avocado_general(self, avocado_default_args=[]):
             rc = cpe.returncode
         return rc
 
+    def _tcinfo(self, testcases, header, logs=True, description=True):
+        if testcases:
```

Comment:
Please add docstring. Would be fine to  mentioned there an example. How does it work and what is needed.

Reply:
I'll add there